src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -14,11 +17,17 @@
 import com.gameList.gamelist.model.Playstation;
 import com.gameList.gamelist.model.Platform;
 import com.gameList.gamelist.model.Xbox;
+import com.gameList.service.INintendoService;
+
+
 
 @Controller
 @RequestMapping("/game-list")
 public class GameListController{
 	
+	@Autowired
```

Comment:
I implemented the NintendoService I created using the @Autowired annotation we learned from class